geco/graphs/utilities.py

```diff
@@ -35,6 +35,73 @@ def edgeweight_properties(graph):
     return max_weight, min_weight, num_of_zero_weights
 
 
+def num_of_simple_cycles(graph):
+    """
+    Finds number of simple cycles
+
+    Parameters
+    ----------
+    graph: nx.Graph
+        Graph to find number of simple cycles in
+
+    Returns
+    -------
+    num_of_simple_cycles: int
+        Number of simple cycles in graph
+
+    Notes
+    -----
+    It was pointed out that this implementation
```

Comment:
This comment is better to be put on the pull request. Also, if you find some input that produces a wrong answer you can add it to the tests so that we are sure that it is fixed before merging.